projects/ngx-valdemort/src/lib/validation-errors.component.spec.ts

```diff
@@ -307,11 +311,13 @@ describe('ValidationErrorsComponent', () => {
   describe('reactive forms', () => {
     let tester: ReactiveComponentTester;
 
-    beforeEach(() => {
-      TestBed.configureTestingModule({});
+    beforeEach(async () => {
+      TestBed.configureTestingModule({
+        providers: [provideExperimentalZonelessChangeDetection(), provideAutomaticChangeDetection()]
```

Comment:
OOC, why not add this to test.ts and avoid repeating it if all tests run the same way?